neevaco/neeva-ios
# Address wallet onboarding feedback

- Import TextEditor is more clickable and makes the border blue when focused
- If there are 12 words in clipboard, we show a Paste & Import button
- Shows the wallet starter screen for web3Mode on install
- Reset matchingCollection when session changes
- Home button is mode dependent now
- Added a dedicated initial suggestion screen for web3Mode

## Checklists

### How was this tested?

### Manual test cases
- Uninstall, reinstall, import a wallet and run through onboarding.
- Click home on both modes
- Check initial suggestions without login on both modes
